src/home.php

```diff
@@ -27,15 +30,214 @@
+                        <select id="selectButton"></select>
+                        <button id="buttonLine">Alle Aufgaben</button>
+                    </div>
+                    <div class="icon" id="icon1">
+                        <i class="fas fa-info-circle fa-2x" style="color: #ffffff; margin-right: 2vw;"> </i>
+                    </div>
+                </div>
+                <div class="overlay" id="overlay1">
+                    <div class="text">Hier kannst du sehen wie du dich über die Zeit verbessert oder verschlechtert hast
+                        bei den einzelnen Aufgaben.
+                    </div>
+                </div>
+            </div>
+            <div class="bar_chart svg-container">
+                <div class="vizButtons">
+                    <div>
+                        <button id="buttonBar1">Wie viele Tests pro Tag
+                        </button>
+                        <button id="buttonBar2">Wie viele Aufgaben
+                            im
+                            Schnitt pro Tag richtig
+                        </button>
+                    </div>
+                    <div class="icon" id="icon2">
+                        <i class="fas fa-info-circle fa-2x" style="color: #ffffff; margin-right: 2vw;"> </i>
+                    </div>
+                </div>
+                <div class="overlay" id="overlay2">
+                    <div class="text">Hier kannst du dich mit allen anderen Schülern vergleichen! Du kannst sehen wie
+                        viele Tests du am Tag absolviert und wie viele Aufgaben du im Schnitt am Tag richtig hattest im
+                        Vergleich zu den anderen.
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="charts">
+            <div class="trafficLight-container"></div>
+            <div class="pie_chart svg-container">
+                <div class="vizButtons">
+                    <div style="margin-left: 18%;">
+                        <span>Alle Tests:</span>
+                        <select id="selectButtonPie"></select>
+                    </div>
+                    <div>
+                        <span>Letzten 5 Tests:</span>
+                        <select id="selectButtonPie5"></select>
+                    </div>
+                    <div class="icon" id="icon3">
+                        <i class="fas fa-info-circle fa-2x" style="color: #ffffff; margin-right: 2vw;"> </i>
+                    </div>
+                </div>
+                <div id="misconceptionAll"
+                     style="display: none; margin: 5% 5% 0 5%;">
+                    <p>Du machst häufig diesen Fehler:</p>
+                    <?php if ($_SESSION['misconceptionAll'] === 0) echo $misconceptionReminder5_0;
+                    elseif ($_SESSION['misconceptionAll'] === 1) echo $misconceptionReminder5_1;
+                    elseif ($_SESSION['misconceptionAll'] === 2) echo $misconceptionReminder5_2;
```

Comment:
```suggestion
                    <?php if ($_SESSION['misconceptionAll'] === 0) echo "<p>Du machst häufig diesen Fehler: $misconceptionReminder5_0 </p>";
                    elseif ($_SESSION['misconceptionAll'] === 1) echo "<p>Du machst häufig diesen Fehler: $misconceptionReminder5_1</p>";
                    elseif ($_SESSION['misconceptionAll'] === 2) echo "<p>Du machst häufig diesen Fehler: $misconceptionReminder5_2</p>";
```

Ggf. den Text nur anzeigen, wenn es einen Fehler zum anzeigen gibt.